### PR TITLE
Feature collision

### LINKED ISSUE #30: Collision detection
- [x] Add checks for when two sprites are on the same place
- [x] Create collision library
- [ ] Update documentation

### PR DESCRIPTION
**Pull request by jaschutte**
Closes #30

### CODE REVIEW

File: lib/collision.cpp [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
+#include "collision.h"
+
+#include <HardwareSerial.h> // REMOVE ME
```
Comment (ItsGers8) [inline lib/collision.cpp]: Remove me?

File: lib/collision.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+#ifndef __COLLISION__
+#define __COLLISION__
+
+#include "gfx.h"
+
+#define MOVEMENT_SPEED 20
+#define MOVEMENT_INTERVAL 150
+
+// Can is a tile standable?
```
Comment (ItsGers8) [inline lib/collision.h]: Maybe change this to a good sentence?

File: lib/collision.cpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+ImageBytes blacklist[] = {
+	GET_IMAGE(road_middle),
+	(ImageBytes)__raw_road_middle_p,
+	0};
+
+uint8_t compare_tile(TileMap *map, Vector2 *tile_pos, RawImage *compare_to)
```
Comment (ItsGers8) [inline lib/collision.cpp:11]: This one isnt documented yet

File: lib/images.h [hunk 4]
```diff
@@ -8,15 +8,6 @@
 // Typecast const uint8_t[] into ImageBytes, which is a costless convertion
```
Comment (ItsGers8) [inline lib/images.h]: CONVERTION GANG

**Review by ItsGers8 — DISMISSED** [review]
Looking good

**Review by jasperdej — APPROVED** [review]

**Pull request merged**